waves/src/index.tsx

```diff
@@ -1,18 +1,16 @@
 import { ChakraProvider } from "@chakra-ui/react";
 import React from "react";
 import ReactDOM from "react-dom";
-import { SSEProvider } from "react-hooks-sse";
 import App from "./App";
+import { BobtimusRateProvider } from "./Bobtimus";
 import "./index.css";
 import reportWebVitals from "./reportWebVitals";
 import theme from "./theme";
 
 ReactDOM.render(
     <React.StrictMode>
         <ChakraProvider theme={theme}>
-            <SSEProvider endpoint="/rate/lbtc-lusdt">
-                <App />
-            </SSEProvider>
+            <BobtimusRateProvider children={<App />} />
```

Comment:
`children` are usually not passed as literal props. You can do this instead:

```tsx
<BobtimusRateProvider>
	<App />
</BobtimusRateProvider>
```

Reply:
Ah, I think now I got it working .